sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java

```diff
@@ -77,6 +77,21 @@ public EditCaseSteps(
       EnvironmentManager environmentManager) {
     this.webDriverHelpers = webDriverHelpers;
 
+    When(
+        "I fill the specific Date of outcome",
```

Comment:
please update the step to reflect the action so QA manual team cand understand the step